### PR TITLE
try to do spinners test

### PR DESCRIPTION
**Pull request by ahospodarysko**

### CODE REVIEW

File: src/test/java/com/khospodarysko/SpinnersTest.java [hunk 1]
```diff
@@ -11,16 +16,37 @@
     private static final Logger logger = LoggerFactory.getLogger(SpinnersTest.class);
 
     @Test
-    public void test() {
+    public void testSpinners() {
         driver.get("file://" + absoluteFilePath("spinners"));
+        WebElement spinnerSmall = driver.findElement(By.cssSelector(".spinner-small"));
```
Comment (kostia-hospodarysko) [inline src/test/java/com/khospodarysko/SpinnersTest.java]: There is a list of small and large spinners.

File: src/test/java/com/khospodarysko/SpinnersTest.java [hunk 2]
```diff
@@ -11,16 +16,37 @@
...
      */
-    public boolean areSpinnersLoaded() {
-        return false;
+    public boolean areSpinnersLoaded(WebDriver driver, WebElement element) {
+        WebDriverWait wait = new WebDriverWait(driver, 20);
+        return wait.until(new ExpectedCondition<Boolean>() {
+            @Override
+            public Boolean apply(WebDriver driver) {
+                try {
+                    return !element.isDisplayed();
```
Comment (kostia-hospodarysko) [inline src/test/java/com/khospodarysko/SpinnersTest.java]: Need to check this for all small and all large spinners, not only for single element.

File: src/test/java/com/khospodarysko/SpinnersTest.java [hunk 3]
```diff
@@ -11,16 +16,37 @@
...
+        Assert.assertTrue(isElementNotDisplayed(spinnerSmall), "Small spinners are disappeared");
+        Assert.assertTrue(isElementNotDisplayed(spinnerLarge), "Large spinners are disappeared");
     }
 
     /**
      * @return true if all spinners have disappeared, false otherwise
      */
-    public boolean areSpinnersLoaded() {
-        return false;
+    public boolean areSpinnersLoaded(WebDriver driver, WebElement element) {
```
Comment (kostia-hospodarysko) [inline src/test/java/com/khospodarysko/SpinnersTest.java]: Pass here two lists: of small and large spinners as two additional params.

**Review by kostia-hospodarysko — CHANGES_REQUESTED** [review]

File: src/test/java/com/khospodarysko/SpinnersTest.java (hunk 2 above)
Comment (ahospodarysko) [reply src/test/java/com/khospodarysko/SpinnersTest.java]: Can i use here operator AND - 
return !(smallSpinner.isEmpty()) && !(largeSpinner.isEmpty())

**Pull request closed without merging**